cont.c

```diff
@@ -1411,13 +1411,13 @@ rb_cont_call(int argc, VALUE *argv, VALUE contval)
  *    end
  *
  *    puts fiber.resume 10
- *    puts fiber.resume 14
- *    puts fiber.resume 18
+ *    puts fiber.resume 1_000_000
```

Comment:
Consider something like this to clarify that no math will be occurring on the second call:

```suggestion
 *    puts fiber.resume :second_return_value
```